src/core/animation-loop.js

```diff
@@ -1,4 +1,4 @@
-/* global window, setTimeout, clearTimeout */
+/* global self, setTimeout, clearTimeout */
```

Comment:
`window` does not exist on worker thread

Reply:
Not familiar with the use of the `self` name on the main thread, though it seems to work in a few browsers I tested. 
* Our `utils/global` module exposes a `window` variable, that is set to `window` in browsers and `global` in node. 
* Perhaps we could set it to `self` if `window` is not available, and then just `import {window} from '../utils/globals'` here?
* Main thought is that it would be pity to break some browser just because of this change.